molecule/core.py

```diff
@@ -198,14 +198,33 @@ def _create_templates(self):
 
         # rakefile
         kwargs = {
-            'molecule_file': self._config.molecule_file,
-            'current_platform': self._provisioner._platform,
+            'state_file': self._config.config['molecule']['state_file'],
             'serverspec_dir': self._config.config['molecule']['serverspec_dir']
         }
         utilities.write_template(
             self._config.config['molecule']['rakefile_template'],
             self._config.config['molecule']['rakefile_file'], kwargs=kwargs)
 
+    def _create_instances_state(self):
```

Comment:
Yup, I gotcha. Just haven't got there yet. This is still a WIP now. Needed better test coverage of group targeting with serverspec and testinfra for this particular change.